### PR TITLE
Fixes

### PR DESCRIPTION
**Pull request by Maks3w**

### CODE REVIEW

File: assets/root-files/phpunit.xml.dist [hunk 1]
```diff
@@ -1,7 +1,7 @@
-<phpunit bootstrap="./test/Bootstrap.php" colors="true">
+<phpunit bootstrap="./Bootstrap.php" colors="true">
```
Comment (weierophinney) [inline assets/root-files/phpunit.xml.dist:1]: No; the phpunit.xml.dist file is in the root, and the test bootstrap is under `test/`. The original was correct.

File: bin/tree-filter.php [hunk 2]
```diff
@@ -115,11 +115,10 @@
...
 // Test directory files
-copy($assetDir . '/test-files/gitignore', 'test/.gitignore');
 file_put_contents(
     'test/Bootstrap.php',
     str_replace('{COMPONENT}', $normalized, file_get_contents($assetDir . '/test-files/Bootstrap.php'))
 );
-copy($testConfigDist, 'test/TestConfiguration.php.dist');
-copy($testConfigTravis, 'test/TestConfiguration.php.travis');
+copy($testConfigDist, 'TestConfiguration.php.dist');
+copy($testConfigTravis, 'TestConfiguration.php.travis');
```
Comment (weierophinney) [inline bin/tree-filter.php:123]: No. I did this for a reason. I wanted the test assets (bootstrap, test configuration) in the `/test` root, which is why I pushed the component tests one level down. I really don't want the bootstrap or test configuration files under the root; `phpunit.xml.dist` makes sense there to simplify running the tests, but it acts like the other dot-files in that regard.

File: bin/composer-rewriter.php [hunk 3]
```diff
@@ -37,7 +37,7 @@
 // Setup development autoloading rules
 $composer['autoload-dev'] = [
     'psr-4' => [
-        'ZendTest\\' . $component . '\\' => 'test/' . $component . '/',
+        'ZendTest\\' . $component . '\\' => 'test/',
```
Comment (weierophinney) [inline bin/composer-rewriter.php:40]: See my note on the tree-filter script.

**Comment by Maks3w** [comment]
IMO src and test should follow the same directory structure. So or both to have a subdirectory with the component name (redundant) or both should have the files directly in the src/test directory

**Comment by weierophinney** [comment]
The problem is that we need the test bootstrap and test configuration, and the concern I have is with mixing those assets with the actual unit test classes.

I saw your recommendation that we move the test configuration into the phpunit.xml.dist as well, but that poses new problems, as it requires changes to the unit tests themselves. Since we're trying to keep history, that raises an issue with older versions of the tests no longer being executable.

If you can solve that problem, I'll happily make the directory structure change, as we can rename the `Bootstrap.php` file to `bootstrap.php` to avoid naming collisions.

**Pull request closed without merging**